website/components/merch-desktop-client/index.jsx

```diff
@@ -2,31 +2,70 @@ import s from './merch-desktop-client.module.css'
 import InlineSvg from '@hashicorp/react-inline-svg'
 
 export default function MerchDesktopDownload({ version, releases }) {
-  const { builds } = releases.versions[version]
+  var { builds } = releases.versions[version]
```

Comment:
```suggestion
  const { builds } = releases.versions[version]
```

nit -- I think this will keep the linter happy

Reply:
Oh yeah, good call thank you my dude, I changed to var as I was pushing some test data to this.